### PR TITLE
Ensure finetune is random shuffled, finetune using input_id.

### PR DESCRIPTION
**Pull request by ranieri**

### CODE REVIEW

File: asmtransformers/asmtransformers/datasets/sentencelabel.py [hunk 1]
```diff
@@ -3,34 +3,12 @@
 from operator import itemgetter
 
 from datasets import Dataset
-from sentence_transformers import InputExample
 from torch.utils.data import IterableDataset
 from tqdm import tqdm
 
 
 class LazySentenceLabelDataset(IterableDataset):
-    """A lazy-loading version of sentence_transformers.datasets.SentenceLabelDataset designed for use with Hugging
```
Comment (fleurvanl) [inline asmtransformers/asmtransformers/datasets/sentencelabel.py:12]: Where did the extensive docstring go? Is al this information no longer correct?

File: asmtransformers/scripts/finetune.py [hunk 2]
```diff
@@ -87,7 +127,7 @@ def eval_triplets(functions):
             yield InputExample(texts=[example['anchor'], example['pos'], example['neg']])
 
     test_functions = functions['test']
-    dev_evaluator = TripletEvaluator.from_input_examples(eval_triplets(test_functions))
+    dev_evaluator = OldFitScalarEvaluator(TripletEvaluator.from_input_examples(eval_triplets(test_functions)))
```
Comment (fleurvanl) [inline asmtransformers/scripts/finetune.py:137]: This might be worth a comment. Do I understand correctly that we tripletify test_functions using eval_triplets; then push that into TripletEvaluator.from_input_examples in order to... reformat? and then push that result into OldFitScalarEvaluator to pick the correct metric? But where do the metrics get calculated?

**Review by fleurvanl — APPROVED** [review]

File: asmtransformers/scripts/finetune.py (hunk 2 above)
Comment (ranieri) [reply asmtransformers/scripts/finetune.py:137]: yes :P

The evaluator is called by the Trainer during old_fit.
